.github/workflows/reusable-php-static-analysis.yml

```diff
@@ -0,0 +1,95 @@
+##
+# A reusable workflow that runs PHP Static Analysis tests.
+##
+name: PHP Static Analysis
+
+on:
+  workflow_call:
+    inputs:
+      php-version:
+        description: 'The PHP version to use.'
+        required: false
+        type: 'string'
+        default: 'latest'
```

Comment:
I'm thinking this is why CI fails despite passing locally - I'll baselined on v8.3.